### PR TITLE
Add relative links to the convert template file

### LINKED ISSUE #1725: When using the Convert feature, some links are not relative to the base url
**Is your request related to a problem?**

When using `markbind init -convert` I realized that some of the relative links do not have a baseUrl but others do.

No baseUrl:
image

With baseUrl:
image

This may cause issues when the user tries to deploy the webpage with the `deploy` feature some of the styles and links may not work as intended.

**Describe the solution you'd like**

Either add  `{{baseUrl}}` to everything or remove it everywhere to keep things consistent.

I think adding baseUrl to everything makes more sense from a deployment perspective as that seems to be the main use case for people converting.

### PR DESCRIPTION
**Pull request by ong6**
**What is the purpose of this pull request?**

- [ ] Documentation update
- [x] Bug fix
- [ ] Feature addition or enhancement
- [x] Code maintenance
- [ ] Others, please explain:

Fixes #1725

**Overview of changes:**
This PR directly address the issue raised in #1725.

I fix it by adding the {{baseUrl}} into the template nunjucks file. 

**Anything you'd like to highlight / discuss:**

**Testing instructions:**

Run `markbind init -c` on a test repo and see that the {{baseUrl}} is indeed there.

image

**Proposed commit message: (wrap lines at 72 characters)**

Add relative links to template file

---

**Checklist:** :ballot_box_with_check:

- [ ] Updated the documentation for feature additions and enhancements
- [ ] Added tests for bug fixes or features
- [x] Linked all related issues
- [x] No unrelated changes

### CODE REVIEW

File: packages/core/src/Site/siteConvertLayout.njk [hunk 1]
```diff
@@ -1,14 +1,14 @@
 <head-bottom>
-  <link rel="stylesheet" href="{{baseUrl}}/stylesheets/main.css">
+  <link rel="stylesheet" href="{% raw%}{{baseUrl}}{% endraw %}/stylesheets/main.css">
```
Comment (jonahtanjz) [inline packages/core/src/Site/siteConvertLayout.njk:2]: Trivia suggestion, able to add some spaces here? Same for the rest :)
```suggestion
<link rel="stylesheet" href="{% raw %}{{ baseUrl }}{% endraw %}/stylesheets/main.css">
```

**Review by jonahtanjz — COMMENTED** [review]
Thanks @ong6 for discovering the issue and fixing it 👍 

Overall looks good. I believe we can also add `{% raw %} {% endraw %}` to `{{ MarkBind }}` in the [footer](https://github.com/MarkBind/markbind/blob/18ef308a43ef5fb404c021f2ae44c4fdbe98b86e/packages/core/src/Site/siteConvertLayout.njk#L44) as well since it's the same issue.

**Comment by ong6** [comment]
@jonahtanjz Thanks for the tips! Will add them!

**Review by ryoarmanda — APPROVED** [review]
LGTM!

**Review by jonahtanjz — APPROVED** [review]
LGTM 👍

**Pull request merged**